src/electron/emain.ts

```diff
@@ -316,81 +322,71 @@ function createMainWindow(clientData: ClientDataType | null) {
         if (win.isFocused()) {
             wasActive = true;
         }
-        if (input.type != "keyDown") {
+        if (waveEvent.type != "keyDown") {
             return;
         }
-        let mods = getMods(input);
+        let mods = getMods(waveEvent);
         if (checkKeyPressed(waveEvent, "Cmd:t")) {
-            win.webContents.send("t-cmd", mods);
-            e.preventDefault();
+            win.webContents.send("electron-key-press", mods);
```

Comment:
checks if we are using this key and then sends "electron-key-press" event instead of individual event